public/js/location.js

```diff
@@ -0,0 +1,139 @@
+(function () {
```

Comment:
I have used 3rd party API for fetching states and cities for US. 
If anyone is facing an issue on their local then let me know.